composer-dependencies.json

```diff
@@ -7,7 +7,8 @@
     "krokedil/klarna-express-checkout": "^1.4.0",
     "krokedil/klarna-onsite-messaging": "^1.3.2",
     "krokedil/settings-page": "^1.1.0",
-    "krokedil/sign-in-with-klarna": "^1.0.6"
+    "krokedil/sign-in-with-klarna": "^1.0.6",
+    "krokedil/klarna-order-management": "^1.1.0"
```

Comment:
Issue: We will need to release a version before it can be added like this. Currently there is no version 1.0.0 of the package yet either, so lets release that before merging this and make sure the composer file is correctly pointing to the version we should use